src/main/scala/rocket/CSR.scala

```diff
@@ -58,9 +57,9 @@ class DCSR extends Bundle {
   val stopcycle = Bool()
   val stoptime = Bool()
   val cause = UInt(width = 3)
+  // TODO: debugint is not in the Debug Spec v13
```

Comment:
Yeah. I had removed it, then put it back because I had a thought of how to use it in the Debug ROM, then realized that idea wouldn't work... so I left it in rather than removing it again in case I thought of another great idea for it. But I think the idea if we did want to use it in the ROM would be to put it in a different implementation specific CSR.